frontend/src/components/activity/content/StoryboardRow.vue

```diff
@@ -0,0 +1,188 @@
+        "
+        :single-line="variant === 'default'"
+        :fieldname="`data.sections[${itemKey}].column1`"
+        :disabled="layoutMode || disabled"
+        :filled="layoutMode"
+      />
+    </td>
+    <td class="e-storyboard-row__text">
+      <api-richtext
+        :label="
+          variant === 'dense'
+            ? $tc('contentNode.storyboard.entity.section.fields.column2Html')
+            : null
+        "
+        :fieldname="`data.sections[${itemKey}].column2Html`"
+        rows="4"
+        :disabled="layoutMode || disabled"
+        :filled="layoutMode"
+      />
+    </td>
+    <td class="e-storyboard-row__responsible">
+      <api-text-field
+        :label="
+          variant === 'dense'
+            ? $tc('contentNode.storyboard.entity.section.fields.column3')
+            : null
+        "
+        :single-line="variant === 'default'"
+        :fieldname="`data.sections[${itemKey}].column3`"
+        :disabled="layoutMode || disabled"
+        :filled="layoutMode"
+      />
+    </td>
+    <td v-if="!layoutMode && !disabled" class="e-storyboard-row__controls">
+      <div class="e-storyboard-row__controls__move">
+        <v-btn
+          v-if="variant === 'dense'"
+          icon
+          small
+          :disabled="isLastSection"
+          @click="$emit('moveUp', itemKey)"
+        >
+          <v-icon>mdi-arrow-up</v-icon>
+        </v-btn>
+        <v-btn
+          v-if="variant !== 'dense'"
+          icon
+          small
+          class="drag-and-drop-handle"
+          :disabled="isLastSection"
+        >
+          <v-icon>mdi-drag</v-icon>
+        </v-btn>
+        <v-btn
+          v-if="variant === 'dense'"
+          icon
+          small
+          :disabled="isLastSection"
+          @click="$emit('moveDown', itemKey)"
+        >
+          <v-icon>mdi-arrow-down</v-icon>
+        </v-btn>
+      </div>
+      <dialog-remove-section @submit="$emit('delete', itemKey)">
+        <template #activator="{ on }">
+          <v-btn
+            icon
+            small
+            class="e-storyboard-row__delete"
+            color="error"
+            :disabled="isLastSection"
+            v-on="on"
+          >
+            <v-icon>mdi-delete-outline</v-icon>
+          </v-btn>
+        </template>
+      </dialog-remove-section>
+    </td>
+  </tr>
+</template>
+<script>
+import DialogRemoveSection from './StoryboardDialogRemoveSection.vue'
+
+export default {
+  name: 'StoryboardRow',
+  components: { DialogRemoveSection },
+  props: {
+    isLastSection: { type: Boolean, required: true },
+    itemKey: { type: String, required: true },
+    layoutMode: { type: Boolean, required: true },
+    disabled: { type: Boolean, default: false },
+    variant: { type: String, default: 'default' },
+  },
+}
+</script>
+<style scoped lang="scss">
+.e-storyboard-row {
+  &:hover .e-storyboard-row__delete {
+    opacity: 1;
+  }
+}
+
+.e-storyboard-row__delete {
+  color: rgba(0, 0, 0, 0.54) !important;
```

Comment:
do we need the important here?

Reply:
Yes, I'm overwriting the VBtn behaviour (which features important as well)